app/controllers/admin/members_controller.rb

```diff
@@ -14,13 +17,24 @@ def destroy
       redirect_to admin_members_path
     end
 
+    def edit
+      @member = Member.find_by!(slug: params[:slug])
+    end
+
+    def update
+      @member = Member.find_by!(slug: params[:slug])
+      @member.update(roles: Role.where(id: params.require(:member).require(:role_ids)))
+
+      respond_with @member, location: admin_members_path
+    end
+
     private
 
     def search_term
       params[:q]
     end
 
-    def auth!
+    def admin?
```

Comment:
I feel like require_admin! or similar (retaining the !) is a bit more informative (this will explode and spray exceptions everywhere if it fails); while I'd expect a is_admin? type method to return a bool.

Reply:
i agree. I'll fix this